taxi/views.py

```diff
@@ -1,3 +1,19 @@
+from lib2to3.fixes.fix_input import context
+
+from django.http import HttpRequest, HttpResponse
 from django.shortcuts import render
 
-# Create your views here.
+from .models import Driver, Manufacturer, Car
+
+
+def index(request: HttpRequest) -> HttpResponse:
+    num_drivers = Driver.objects.count()
+    num_manufacturers = Manufacturer.objects.count()
+    num_cars = Car.objects.count()
+    local_context = {
+        "num_drivers": num_drivers,
+        "num_manufacturers": num_manufacturers,
+        "num_cars": num_cars
+    }
+
+    return render(request, "taxi/index.html", context=local_context)
```

Comment:
The `index` view function is not documented. It's a good practice to add docstrings to your functions to describe their purpose, parameters, and return values. This helps other developers understand your code better.